packages/react-components/src/menu/src/Menu.tsx

```diff
@@ -20,66 +23,79 @@ import {
 import { MenuContext } from "./MenuContext";
 import { MenuItem } from "./MenuItem";
 import { MenuSection } from "./MenuSection";
-import { NodeType, useCollection } from "../../collection";
 import { SelectionMode } from "./selectionMode";
-import { any, arrayOf, bool, elementType, func, number, object, oneOf, oneOfType, string } from "prop-types";
-import { forwardRef } from "react";
 import { isNil, isNumber } from "lodash";
 
 export const ItemKeyProp = "data-o-ui-key";
 
-const propTypes = {
+export interface InnerMenuProps {
+    /**
+     * @ignore
+     */
+    id?: string;
     /**
      * A controlled set of the selected item keys.
      */
-    selectedKeys: arrayOf(string),
+    selectedKeys?: string[]
     /**
      * The initial value of `selectedKeys` when uncontrolled.
      */
-    defaultSelectedKeys: arrayOf(string),
+    defaultSelectedKeys?: string[]
     /**
      * Called when the selected keys change.
      * @param {SyntheticEvent} event - React's original SyntheticEvent.
      * @param {String[]} keys - The keys of the selected items..
      * @returns {void}
      */
-    onSelectionChange: func,
+    onSelectionChange?(event: SyntheticEvent, keys: string[]): void
     /**
      * The type of selection that is allowed.
      */
-    selectionMode: oneOf(["none", "single", "multiple"]),
+    selectionMode?: SelectionMode;
```

Comment:
Does it render well in Storybook props table?

Reply:
![image](https://user-images.githubusercontent.com/38871812/114090332-47f14a80-9885-11eb-9727-2da5bcc6fd85.png)